mail/api_test.py

```diff
@@ -40,6 +40,19 @@ def test_from_address(self, mock_post):
         _, called_kwargs = mock_post.call_args
         assert called_kwargs['data']['from'] == 'mailgun_from_email@example.com'
 
+    @override_settings(EMAIL_SUPPORT='mailgun_from_email@example.com')
+    def test_from_address_with_sender_name(self, mock_post):
+        """
+        Test that the sender name and 'from' address for our emails is set correctly.
+        """
+        sender = 'tester'
+        MailgunClient.send_bcc('email subject', 'email body', ['will_be_ignored@example.com'], sender_name=sender)
+        _, called_kwargs = mock_post.call_args
+        self.assertEqual(
+            called_kwargs['data']['from'],
+            '{sender_name} <mailgun_from_email@example.com>'.format(sender_name=sender)
+        )
+
```

Comment:
This test is not enough: given that you added an extra argument to many functions, you should test all the modified functions with the extra argument.